muxplore/main.c

```diff
@@ -261,6 +261,7 @@ void reset_label_long_mode() {
 }
 
 void set_label_long_mode() {
+    lv_task_handler();
```

Comment:
the lv_task_handler was enough to get the label to be up to date and seems to be way more responsive.  This allowed me to remove the full screen refresh (lv_refr_now) all together